README.md

```diff
@@ -89,9 +89,9 @@ Parameters:
 
 The `click` endpoint captures any click interaction with the tile. This includes `click`, `block`, `pin`, `unpin`, `sponsored`, and `sponsored_link`.
 
-When the user interacts with the tile, an asynchronous ping is sent to Mozilla servers. This is to measure a click event. Since our business model is Cost per Click (CPC), this metric is vital.
+When the user interacts with the tile, an asynchronous ping is sent to Mozilla servers. This is to measure a click event. **Since our business model is Cost per Click (CPC), this metric is vital**.
```

Comment:
I think the comment about how CPC was our business model didn't turn out to be true. We can remove that line.

What is our business model now? Do the right thing for the user.